github/actions_artifacts.go

```diff
@@ -121,6 +121,10 @@ func (s *ActionsService) DownloadArtifact(ctx context.Context, owner, repo strin
 	}
 
 	parsedURL, err := url.Parse(resp.Header.Get("Location"))
+	if err != nil {
+		return nil, nil, err
```

Comment:
Please return a copy of the previous response here, as in line 120 above.

```suggestion
		return nil, newResponse(resp), err
```

Reply:
- [x] Done.